### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by neuromancer** [acoto87/war1#1]
Hi,

I'm following this neat re-implementation engine. The screenshots look great so far, but it is a little unclear from the README what elements from gameplay are implemented (there is a TODO list, but it is hard to follow, if you are not working in the code). Can you add a few lines in the README on what to expect if you want to test this engine?

Thanks!

**Comment by acoto87** [comment]
Hi, thanks for the interest in the project.

I know the TODO list is messy and doesn't provide a clear current status. It's meant for my development, but maybe I could clean it a little.

I will update the README file soon, with a clear current status and how to build the engine and these kind of stuff.

Currently you can play the first two missions of each campaign. I will also release an alpha or pre-alpha version soon.

Current features (this list is not complete):
 * Path finding and collisions detection
 * Unit movement with animations
 * Unit training
 * Building construction
 * Resources gathering by peasants and peons
 * Features specification for maps (this is specify if the player can train, build or research certain units, buildings or spells)
 * Win/lose condition checking
 * Fog of war
 * Basic behavior for enemies (it will attack your units if you get near enough, it will chase your units)

Still to develop (this list is not complete):
 * AI (I'm planning to do a basic one first and build from that more complex ones)
 * Cinematics (I've the code for reading FLIC files just need to integrate it into the engine)
 * Typing commands for cheats and other stuff that could be activated by it in the engine
 * Save/Load

There are also several bugs that I need to fix, those are more clear in the Bugs section of the TODO file. I'm working on it every day (yesterday I just fix an audio bug) and push the changes every day too.

**Comment by neuromancer** [comment]
Thanks for the update!, can you also comment in the README about other Warcraft 1 reimplementations, so people can understand better how implemented is this one?

**Comment by acoto87** [comment]
Sure, I'll do that in the following days. Stay tuned.

**Comment by jerkstore369** [comment]
This is really cool I just found out about it by accident. I'd love to see it be as close to the original as possible but with quality of life improvements i.e. simply right clicking to tell a unit where to go instead of all those ridiculous hotkeys I remember it had. I used to love the wizard that can create elementals. Anyways, thanks for this.

**Comment by acoto87** [comment]
Thanks @jerkstore369!

Yes, I'm aiming to have the game with those quality improvements without modifying too much the original behavior of the game. Right click to move, attack or gather was a day 1 implementation. It also have other things like move through one or more way points and buildings no longer have to be built next to a street.

I've the current state of the game published here btw, in case you want to try it :) https://acoto87.itch.io/war1

Best regards

**Comment by KuriKai** [comment]
Any chance of being able to build on amd64 linux/ubuntu?

**Comment by acoto87** [comment]
Yes @KuriKai, you should be able to build it on Linux. Sadly I haven't been able to install a VM with Linux to test that, but at the minute I can, I'll make a build for that. The closest build on Linux is that I've is in Raspbian on a Raspberry PI where I test the project, and it runs fine there. Keep me posted if you decided to try it yourself. In any case I'll let a note here when I can do it myself.

**Comment by KuriKai** [comment]
I managed to get it compiled, and seems to run fine. :)

I'll make a clone and try push my changes
